### PR TITLE
DCT-282

### PR DESCRIPTION
**Pull request by Craig-Elre**
Add an example of an issue and how to fix it for `RE0069`

### CODE REVIEW

File: hs/t/y/underscore_unread.rsh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'reach 0.1';
```
Comment (TheChronicMonster) [inline hs/t/y/underscore_unread.rsh:1]: Let's generate the output and add that to the PR, as well.

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

File: docs/src/rsh/errors/index.md [hunk 2]
```diff
@@ -2236,7 +2248,7 @@ a `{!rsh} Participant` or `{!rsh} ParticipantClass`.
 For example, the program below erroneously uses `{!rsh} Anybody.publish()` without declaring any `{!rsh} Participant` or `{!rsh} ParticipantClass`:
 
 ```reach
-load: /hs/t/n/Err_No_Participants.rsh
+load: ./hs/t/n/Err_No_Participants.rsh
```
Comment (jeapostrophe) [inline docs/src/rsh/errors/index.md:2251]: Doesn't this need to be changed?

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/rsh/errors/index.md (hunk 2 above)
Comment (jeapostrophe) [reply docs/src/rsh/errors/index.md:2251]: Maybe you need to rebase?

Comment (Craig-Elre) [reply docs/src/rsh/errors/index.md:2251]: Fixing it caused issues with merging for Hamir(who already had a fix), so I switched it back.

Comment (jeapostrophe) [reply docs/src/rsh/errors/index.md:2251]: Right now, if we were to merge yours, then it would be wrong

Comment (Craig-Elre) [reply docs/src/rsh/errors/index.md]: Fixed it again

**Pull request merged**